source/resource/qx/tool/cli/templates/loader/loader-node.tmpl.js

```diff
@@ -1,6 +1,22 @@
 (function(){ 
 
   var path = require("path");
+  var module = require("module").Module;
+
+  var appModulePaths = [];
+  var old_nodeModulePaths = module._nodeModulePaths;
```

Comment:
is `_nodeModulePaths` an undocumented node feature?

Reply:
found it here: https://www.npmjs.com/package/app-module-path
It's an exported function